samples/drop-ins/SampleDropIn/DropIn.cs

```diff
@@ -21,4 +23,18 @@ public async ValueTask ConfigureAsync(IServiceProvider serviceProvider, Cancella
         var activityRegistry = serviceProvider.GetRequiredService<IActivityRegistry>();
         await activityRegistry.RegisterAsync<SampleActivity>(cancellationToken: cancellationToken);
     }
+
+    public void Unconfigure(IServiceProvider serviceProvider)
+    {
+        var logger = serviceProvider.GetRequiredService<ILogger<DropIn>>();
+        try
```

Comment:
I'm wondering if it would be better for the calling code to do the try/catch stuff? That way, we don't have to repeat this exception handling in other drop-ins.

Reply:
I agree with you, I'm going to try and implement this on the calling code. 
I think I made it like this on the first place because the exception was not being caught by the calling code because of it being dynamically loaded, but I'm most likely wrong.
I'll come back with updates in a few minutes.

Reply:
Resolved on latest commit.